pants.toml

```diff
@@ -85,12 +85,23 @@ root_patterns = [
 ]
 
 [python]
+# resolver_version is always "pip-2020-resolver". legacy is not supported.
```

Comment:
Our current infra relies on the legacy resolver and old versions of pip. Old versions of pants also supported the older pip + legacy resolver, but the version we're using now does not.

So, I left this comment here for future reference and to call out this departure from how we used to do things.